app.js

```diff
@@ -26,17 +28,23 @@ app.use((request, response) => {
 });
 
 app.use((err, request, response, next) => {
-  // console.log(err, "err in app");
+  console.log(err,"err in app");
```

Comment:
We should remove console.logs, would be alright to have one in the 500 error handler as a last resort 🙂